### ISSUE TITLE
BigQuery Client.Query() errors for long-running queries

### ISSUE THREAD

**Issue by magnalite** [yoshidan/google-cloud-rust#174]
If you use `Client.Query(...)` for a query which can take a while (>~10 seconds), BigQuery will respond with:
```
{
  "kind": "bigquery#queryResponse",
  "jobReference": {
    "projectId": "...",
    "jobId": "...",
    "location": "US"
  },
  "jobComplete": false
}
```

Currently it is expected this is a `google_cloud_bigquery::http::job::query::QueryResponse` but it fails to parse due to not containing every field expected and there is also no retry logic to recheck and await job completion.

**Comment by yoshidan** [comment]
Thank you for your report. I will fix it soon.

**Comment by yoshidan** [comment]
I have fixed and published as v0.1.1. Please try it.

**Comment by magnalite** [comment]
This appears to work for me now! Thanks for fixing this so fast! Your gcp libraries are awesome.
